src/lighteval/tasks/lighteval_task.py

```diff
@@ -517,6 +483,22 @@ def construct_requests(
                 )
                 for i, choice in enumerate(formatted_doc.choices)
             ]
+
+        if self.has_metric_category[MetricCategory.MULTICHOICE_PMI]:
+            assert (
```

Comment:
Why do you add Multichoice_PMI 2 times ? Do you need to create 2 requests ?

Reply:
I do need to create two requests one with the `uncoditioned_query` and one with normal query

Reply:
OK!